src/test/ui/span/lint-unused-unsafe.thir.stderr

```diff
@@ -0,0 +1,1824 @@
+error: unnecessary `unsafe` block
+  --> $DIR/lint-unused-unsafe.rs:1054:29
+   |
+LL |             let _ = async { unsafe {
+   |                             ^^^^^^ unnecessary `unsafe` block
+   |
```

Comment:
This isn't pointing at the reason, the `async fn` in 1051. Or is it because there's another `unsafe` block within it before calling the `unsafe fn`? Either way, we should be pointing at the *reason* this is unnecessary, otherwise people will have to either already know what the problem is or start changing things at random.